static/panes/ast-view.js

```diff
@@ -150,7 +150,7 @@ Ast.prototype.getDefaultPaneName = function () {
 };
 
 Ast.prototype.getPaneTag = function () {
-    if(this._editorid !== false) {
+    if(this._editorid !== undefined) {
```

Comment:
I believe this will cause an issue with the "Editor #false" bug, the UI code does actually pass `false` rather than `undefined` when in tree mode

Reply:
As discussed in Discord I'm updating these to all be `if (this._editorid)` etc - please take another look when you get a chance! Thanks again for your patience whlie I learn all this new stuff :-)